src/Symfony/Component/HttpFoundation/ServerBag.php

```diff
@@ -89,6 +89,7 @@ public function getHeaders()
 
         // PHP_AUTH_USER/PHP_AUTH_PW
         if (isset($headers['PHP_AUTH_USER'])) {
+            $headers['PHP_AUTH_PW'] = $headers['PHP_AUTH_PW'] ?? '';
```

Comment:
let's not store the empty string as suggested by stof.
using `($headers['PHP_AUTH_PW'] ?? '')` on the line below should be enough

Reply:
Done